pkgs/top-level/all-packages.nix

```diff
@@ -14793,11 +14793,11 @@ in
 
   inherit (callPackages ../development/libraries/libressl { })
     libressl_3_0
-    libressl_3_1;
```

Comment:
LibreSSL 3.1 on the other hand, will be supported for one year after [the OpenBSD 6.7 release](https://www.openbsd.org/67.html), so it is still supported. In the past we’ve kept both of the supported versions, and then made `libressl` an alias for the latest one.

On the other hand, nothing is explicitly referencing 3.1 at this time, so we could also include only the latest one, I’m not sure if there is a policy on this?

Reply:
Regarding 3.1, LibreSSL 3.2 may or may not introduce incompatibilities with its consumers, so I think it is convenient to keep it until at least EOL to address such issues.

Reply:
Done.